### PR TITLE
Ec2 docker sizes and nginx routing issues

### PR DESCRIPTION
**Pull request by zimba-github**
https://app.clickup.com/t/86b2rhq6v

t2.micro hangs when all the FastAPI containers are run on a single instance. Due to the time already spent debugging the docker-compose issue, these containers will be run individually. At a later time we'll debug the docker-compose issue.

### CODE REVIEW

File: scripts/pull-images-from-ecr-on-ec2.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+# List of images to build and push
+IMAGES=(
+  "zimba-rei-common-components_deals-api"
+#  "zimba-rei-common-components_investor-profiles-api"
+#  "zimba-rei-common-components_listings-api"
+#  "zimba-rei-common-components_mortgages-api"
+#  "zimba-rei-common-components_nginx"
+#  "zimba-rei-common-components_subscriptions-api"
+#  "zimba-rei-common-components_underwritings-api"
```
Comment (powermukisa) [inline scripts/pull-images-from-ecr-on-ec2.sh:16]: should these be commented out?

File: docker-compose-ec2-no-nginx.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+version: '3.8'
+
+services:
```
Comment (powermukisa) [inline docker-compose-ec2-no-nginx.yml:3]: maybe add to the readme that this docker-compose is not in use on the ec2

**Review by powermukisa — APPROVED** [review]
LG%C

File: scripts/pull-images-from-ecr-on-ec2.sh (hunk 1 above)
Comment (zimba-github) [reply scripts/pull-images-from-ecr-on-ec2.sh:16]: Not finally. These scripts are here just for convenience and are not yet used in CI/CD. I will update them when we get to automating the process.

**Pull request merged**